src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(result, number) {
```

Comment:
will be better to name variables in more general name
it's not clear why one of arguments is the result and the other is not